src/client-apps/nuxeo-mobile/how-to-customize-nuxeo-mobile-workflow-tasks-layout.md

```diff
@@ -0,0 +1,333 @@
+---
+title: 'HOWTO: Customize Nuxeo Mobile Workflow Tasks Layout'
+review:
+    comment: ''
+    date: ''
+    status: ok
+details:
+    howto:
+        excerpt: This how-to explains how to customize Nuxeo Mobile workflows task layouts.
+        level: Intermediate
+        tool: JSON Extention
+        topics: Nuxeo Mobile
+labels:
+    - nuxeo-mobile
+    - yachour
+    - howto
+
+toc: true
+
+---
+
+Since the version 1.13.0 on iOS and 1.11.0 on Android, workflows defined in studio can be processed by users from the mobile application. 
+
+To enable the processing of tasks from mobile, there is an additional step to this configuration, to define the layout of these tasks on mobile devices : 
```

Comment:
To enable the processing of tasks on mobile devices, the tasks mobile layout need to be defined: